docs/app/monitoring-pelican-services/prometheus/page.mdx

```diff
@@ -54,32 +54,77 @@ This yields the duration in seconds.
   | `registry`   | Namespace registration at the Registry | Origin and cache servers     |
   | `director`   | Object transfer tests from the Director | Origin and cache servers     |
   | `topology`    | Data fetch from the OSDF topology server                 | All servers (OSDF mode only)                  |
+  | `IO-concurrency` | IO concurrency limit | Origin and cache servers |
+  | `prometheus`   | Prometheus server | All servers |
+  | `config-updates` | Authfile and scitokens.cfg updates | Origin and cache servers |
```

Comment:
I don't find these descriptions particularly helpful. For example, if the `config-updates` component is critical, exactly what does that mean if I don't already know what the metric is for? Can these be fleshed out a bit?